jdaviz/configs/imviz/plugins/catalogs/catalogs.vue

```diff
@@ -28,6 +28,24 @@
       <g-file-import id="file-uploader"></g-file-import>
     </plugin-file-import-select>
 
+    <v-row v-if="catalog_selected === 'Gaia'">
+      <j-docs-link>
+        See the <j-external-link link='https://astroquery.readthedocs.io/en/latest/gaia/gaia.html' linktext='astropy.gaia docs'></j-external-link> for details on other configurable features.
```

Comment:
but do we allow for passing any of those optional arguments?

Reply:
We do not, but maybe a particularly knowledgeable user could use the astroquery api to enable them?